src/database/DataTypes/TransactionBatch.js

```diff
@@ -35,6 +35,18 @@ export class TransactionBatch extends Realm.Object {
     }
   }
 
+  /**
+   * Returns either the prescriber name who prescribed the medicine
+   * to a patient, or an empty string.
+   * @return {String}
+   */
+  get prescriber() {
+    const { prescriber } = this.transaction;
+    const { firstName = '', lastName = '' } = prescriber || {};
+    if (!firstName && !lastName) return '';
+    return `${firstName} ${lastName}`;
```

Comment:
perhaps a `trim` to cater for the *very* unlikely noFirstName scenario:

```suggestion
    return `${firstName} ${lastName}`.trim();
```

Reply:
nice! be6c4307f5d8db21ef83011666bcd7fb47892445